kubernetes/windows/main.ps1

```diff
@@ -649,6 +649,15 @@ function Get-ContainerRuntime {
 
 function Start-Fluent-Telegraf {
 
+    Set-ProcessAndMachineEnvVariables "TELEMETRY_CUSTOM_PROM_MONITOR_PODS" "false"
```

Comment:
does this mean that even SIDECAR_SCRAPING_ENABLED set as disabled, those variables will be set?